modules/navier_stokes/doc/content/source/variables/ExtrapolateAtJumpFacesVariable.md

```diff
@@ -0,0 +1,16 @@
+# ExtrapolateAtJumpFacesVariable
+
+When this type of variable is used instead of a typical finite volume variable, faces for
+which the neighboring elements have different cell-center values will be treated as
```

Comment:
Might be mentioning what different means in this case. I suppose this will be used for porosity so might be worth mentioning the target use case too.

Reply:
Is "unequal" more clear?

Reply:
Hahaha, I just had a typical FV field in my head where we have different values in each cell. But if you rename this guy feel free to disregard this comment.

Reply:
renamed